zerver/webhooks/sentry/view.py

```diff
@@ -67,6 +67,8 @@
 # Maps "platform" name provided by Sentry to the Pygments lexer name
 syntax_highlight_as_map = {
     "go": "go",
+    "java": "java",
+    "javascript": "javascript",
```

Comment:
I think it's probably worth merging this and finding out what formats we see in production and then iterating on this to make those work, but I think Sentry's platforms don't necessarily all use the same format for events in different platforms (which is crazy).  We can address this is a follow-up pass of fixing issues in webhooks; ultimately we should support any platform sentry supports.

Reply:
Those two additions are pulled from values I saw in production.  Knowing how well they would have rendered the data we got from Sentry is another question, though.